addons/mail/static/src/js/many2many_tags_email.js

```diff
@@ -82,7 +82,7 @@ var FieldMany2ManyTagsEmail = M2MTags.extend({
                 res_model: self.field.relation,
                 res_id: resID,
                 context: self.record.context,
-                title: _t("Please complete customer's informations and email"),
+                title: _t(""),
```

Comment:
Why do you want to translate "" ^^

I think that the line could be totally removed ;)